tools/validators/instance_validator/README.md

```diff
@@ -90,40 +96,40 @@ results in these actions:
 
 **NOTE:** The new building configuration format requires that entities are keyed by Version 4 UUIDs (referred to as guids) instead of the code. To convert from old format to the new format, run your building configuration file(.yaml) through the [guid generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator).
 
-### Warnings and Errors
+## Warnings and Errors
 
 The Instance Validator has two types of outputs: **Warnings** and **Errors**. In short, errors are exposed when a Building Config is not readable by the instance validator while warnings are exposed when a Building Config is readable but its contents may not be valid.
 
-#### Errors
+### Errors
 Errors cause the validator to exit prematurely with some kind of exception. For example, An error could be caused by an entity block not containing a `GUID`. This would raise an exception in the validation logic and cause the validator to exit prematurely because the instance validator expects every entity block in a Building Config to have an associated `GUID`. The [GUID generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator) must be used to generate GUIDs for a Building Configuration file. Other entity block elements are `type` and `code`.
 
-    The following entity block would throw an error because it does not contain a type nor a code.
-
-    8318f346-10b4-44f0-ac0b-bf7659510bfa:
-      translation:
-        zone_air_temperature_sensor:
-          present_value: "temp_1"
-          units:
-            key: "units"
-            values:
-              degrees_celsius: "degC"
-              degrees_fahrenheit: "degF"
-
-#### Warnings
+The following entity block would throw an error because it does not contain a type or a code.
+```yaml
```

Comment:
nit: remove "yaml"

Reply:
including `yaml` formats the code block as if it were yaml text